README.md

```diff
@@ -1,70 +1,48 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Bookstore
 
-## Available Scripts
+> Set up a React app using React documentation.
```

Comment:
[OPTIONAL] It would be nicer to add to your Readme a descriptive description of the project telling others what the project is about so that the users know more about your project.